addons/website/static/src/scss/website.scss

```diff
@@ -386,6 +386,14 @@ font[class*='bg-'] {
     @extend %o-we-background-layer-parent;
 }
 
+// Floating elements are not allowed to float at the end of a section
+// so that they are not floating over the next section
+section::after {
+    content: "";
+    display: table;
+    clear: both;
+}
+
```

Comment:
note that we already have the same thing here https://github.com/odoo-dev/odoo/commit/d7acfdae2b2d2feb320cb509b30b67082f80cfcd
Do we still need it ? Can't we combine both ?
cc @qsm-odoo

Reply:
Who could indeed maybe just adapt the rule you mention to target all containers instead of direct children of wrap... can't remember why I did not (or even why BS removed it)